src/components/Header/LanguageSelect.tsx

```diff
@@ -1,9 +1,9 @@
-import type { FunctionalComponent } from 'preact';
+import type { FunctionComponent } from 'preact';
```

Comment:
preact v10 doesn't seem to suggest using `FunctionalComponent ` https://preactjs.com/guide/v10/typescript/#function-components though it's still exported. But maybe I shouldn't include those changes in this pull request?